scripts/gitlint/ncs.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Even this is from NCS, so, should probably be removed.

Reply:
True, will remove those. I thought I had removed all of these downstream files but I missed some.

Reply:
@krish2718 I removed ncs specific stuff but I am not sure if we should keep this commit atm. Would it make sense to remove this commit, at least temporarily, and add it later when it would work as expected. WDYT?

Reply:
Yes, that should be fine

Reply:
Ok, I will remove the CI commit. Lets get it back some later date and the repo is more stable.